pkg/config/configuration.go

```diff
@@ -582,6 +593,18 @@ func (c Configuration) GetAPILoggingSpec() APILoggingSpec {
 	return *c.Spec.LoggingSpec.APILogging
 }
 
+// GetWorkflowSpec returns the Workflow spec.
+// It's a short-hand that includes nil-checks for safety.
+func (c *Configuration) GetWorkflowSpec() WorkflowSpec {
+	if c == nil || c.Spec.WorkflowSpec == nil {
+		return WorkflowSpec{
+			MaxConcurrentWorkflows:  100,
```

Comment:
Default configuration should go in https://github.dev/cgillum/dapr/blob/60401b1cbd009ee20dfc61aa87bfa27f1f18b6a8/pkg/config/configuration.go#L362